content/enterprise_influxdb/v1.9/administration/configure-password-hashing.md

```diff
@@ -45,3 +45,12 @@ The `meta.password-hash` setting must be the same in both the data and meta node
   # Configures strict FIPS-readiness check on startup.
   ensure-fips = true
 ```
+
+## Change password hashing algorithm
+
+To change the password hashing algorithm used by an existing InfluxDB Enterprise cluster:
+
+1. Ensure all meta and data nodes are running InfluxDB Enterprise 1.9.3 or later.
+2. Edit the meta and data node configuration files as in the [example above](#example-configuration).
+2. Restart each meta and node to pick up the configuration change.
+3. Change each password. This will store each password in the new hash format.
```

Comment:
Numbering is 1, 2, 2, 3 ...perhaps 1,2,3,4?

This last instruction isn't a very clear.  Essentially, you are suggesting that for all existing usernames within the cluster, the passwords MUST be updated in order for the new hashing algorithm to be applied.  If the passwords are NOT updated, the old algorithm will continue to be used.

How is an administrator to know whether they have compliance?  Meaning, is there a way to check which algo is in use? Which users need prompting to make changes?